Userland/Utilities/beep.cpp

```diff
@@ -1,13 +1,15 @@
 /*
- * Copyright (c) 2020, the SerenityOS developers.
+ * Copyright (c) 2022, the SerenityOS developers.
```

Comment:
Please make this a range, this file has existed _since_ 2020:
```suggestion
 * Copyright (c) 2020-2022, the SerenityOS developers.
```